config/environments/production.rb

```diff
@@ -87,6 +87,12 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
+  # Log disallowed deprecations.
+  config.active_support.disallowed_deprecation = :log
```

Comment:
I don't think we want deprecation warnings in production logs - can this be set to `:none`?

Reply:
`:none` doesn't appear to be an option, so I changed it to `:silence`